runtime/src/main/resources/nflow-rest-api.properties

```diff
@@ -0,0 +1,9 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: Huawei Inc.
+#
+#
+nflow.rest.cors.enabled=false
```

Comment:
profile can be just 'nflow'

Reply:
@swaroopar Hi Swaroop
 https://github.com/NitorCreations/nflow/blob/535348a22d159ca51e536a861c94ce17854881bb/nflow-rest-api-common/src/main/java/io/nflow/rest/config/NflowRestApiPropertiesConfiguration.java#L7 , This configuration file is the built-in configuration file when springboot integrates nFlow, not a Profile file. If we want to use nFlow, it must be configured. Springhuizidong goes to the classpath to find this file. We cannot use Profile to decide whether to use it or not.

Reply:
@WangLiNaruto then can we skip having this file? We can directly put the properties in the application.properties and add the profile to the default profiles.